content/docs/0.16.x/manage/shipyard/index.md

```diff
@@ -1,290 +1,103 @@
 ---
-title: Shipyard
+title: Working with shipyard files
 description: Information about shipyard, sequences and tasks to define processes and workflows.
 weight: 25
-keywords: [0.16.x-manage]
+keywords: [0.15.x-manage]
 aliases:
 ---
 
-## Declare Shipyard (before creating a project)
-
-* A shipyard is defined at the level of a project. This means that all services in a project share the same shipyard definition.
+A [shipyard.yaml](../../reference/files/shipyard) file is defined at the level of a project.
+This means that all services in a project share the same shipyard definition.
 
 * A shipyard defines the stages each deployment has to go through until it is released in the final stage, e.g., the production stage.
 
-* A shipyard can consist of any number of stages; but at least one. A stage must have at least the name property.
-
-* A stage can consist of any number of sequences; but at least one.
-
-
-### Definition of Stage
+* A shipyard can consist of any number of stages; but at least one. Each stage must have at least the name property.
 
-A stage is declared by its name. This name is used for the branch in the Git repository and Kubernetes namespace to which services at this stage will be deployed.
+* A stage can contain any number of sequences but must have at least one.
 
 **Example of a shipyard with three stages:**
 
-```yaml
-apiVersion: spec.keptn.sh/0.2.3
-kind: "Shipyard"
-metadata:
-  name: "shipyard-sockshop"
-spec:
-  stages:
-    - name: "dev"
-    - name: "hardening"
-    - name: "production"
-```
-
-### Definition of Sequence in a Stage
-
-After defining stages, sequences can be added to a stage. A sequence is an ordered list of tasks that are triggered sequentially. A `sequence` consists of the following properties:
-
-* `name`: A unique name of the sequence
-* `triggeredOn` (optional): An array of events that trigger the sequence.
-* `tasks`: An array of tasks executed by the sequence in the declared order.
+    apiVersion: spec.keptn.sh/0.2.3
+    kind: "Shipyard"
+    metadata:
+      name: "shipyard-sockshop"
+    spec:
+      stages:
+        - name: "dev"
+        - name: "hardening"
+        - name: "production"
 
 **Example:** Extended shipyard with a delivery sequence in all three stage:
 
-```yaml
-apiVersion: spec.keptn.sh/0.2.3
-kind: "Shipyard"
-metadata:
-  name: "shipyard-sockshop"
-spec:
-  stages:
-    - name: "dev"
-      sequences:
-      - name: "delivery"
-        tasks: 
-        - name: "deployment"
-        - name: "release"
-
-    - name: "hardening"
-      sequences:
-      - name: "delivery"
-        triggeredOn:
-          - event: "dev.delivery.finished"
-        tasks: 
-        - name: "deployment"
-        - name: "release"
-
-    - name: "production"
-      sequences:
-      - name: "delivery"
-        triggeredOn:
-          - event: "hardening.delivery.finished"
-        tasks: 
-        - name: "deployment"
-        - name: "release"
-```
-
-### Reserved Keptn Tasks
```

Comment:
Excellent! The first time I've seen this anywhere!

Reply:
Ha!  I actually stole this from https://keptn.sh/docs/0.15.x/manage/shipyard/#reserved-keptn-tasks -- but now we'll know where it is ;-)

Reply:
Note that this was deleted from this file -- it's in the reference page